### ISSUE TITLE
Revealjs slide notes affect plot size

### ISSUE THREAD

**Issue by drcaprosser** [quarto-dev/quarto-cli#2904]
### Bug description

Hello, 

I have noticed that when adding notes to a slide with a plot, the notes seem to shrink the plot, even though the notes are not visible on the slide, which seems non-ideal.

Here's a toy example, I'm using RStudio 2022.07.0 Build 548 on Windows 11:

````
---
title: "Slide notes affect plot size"
format: revealjs
---

## This is a slide without notes

```{r}

library(ggplot2)

data <- data.frame(x = rnorm(100),
                   y = rnorm(100))

ggplot(data, aes(x = x, y = y)) +
  geom_point()

```
## This is a slide with notes

```{r}

library(ggplot2)

data <- data.frame(x = rnorm(100),
                   y = rnorm(100))

ggplot(data, aes(x = x, y = y)) +
  geom_point()

```

::: notes

- This is a note

:::
````
 

### Checklist

- [X] Please include a minimal, fully reproducible example in a single .qmd file? Please provide the whole file rather than the snippet you believe is causing the issue.
- [X] Please [format your issue](https://quarto.org/bug-reports.html#formatting-make-githubs-markdown-work-for-us) so it is easier for us to read the bug report.
- [X] Please document the RStudio IDE version you're running (if applicable), by providing the value displayed in the "About RStudio" main menu dialog?
- [X] Please document the operating system you're running. If on Linux, please provide the specific distribution.

**Comment by jjallaire** [comment]
re-assigning to @cderv as he did the auto-fit feature for reveal. hopefully as easy as just adding another selector to our exclude list.

**Comment by cderv** [comment]
I can't reproduce this on my side. @drcaprosser which version of Quarto are you using ? 

I believe this was supposed to be fixed in https://github.com/quarto-dev/quarto-cli/pull/1489 which should be included as soon as version 1.1.27

**Comment by drcaprosser** [comment]
Two apologies @cderv:

1. I thought I was up to date but I wasn't (and have now learned that update.packages() doesn't pick up the quarto that is bundled with RStudio, which is good to know). I think I was using 0.9.649. I've updated RStudio, which has 1.1.189 and that seems to have solved the problem. 

2. I searched through the previous issues but didn't spot #1489.
